dashboard/dashboard-front/src/utils/time.ts

```diff
@@ -65,3 +65,24 @@ export function timestampSecondsAgo(timestampS: number | undefined): number | nu
     const diffMs = new Date().getTime() - timestampS * 1000
     return Math.floor(diffMs / 1000)
 }
+
+export function formatDuration(timestampStart: number | undefined, timestampEnd: number | undefined): string {
+    if (!timestampStart || !timestampEnd)
+        return ''
```

Comment:
This will return an empty string when any of the input values are `0`. It's supposed to be a timestamp so this probably has no chance of occurrence. Just making sure it's supposed to work that way.